lldb/examples/python/templates/scripted_thread_plan.py

```diff
@@ -0,0 +1,70 @@
+from abc import abstractmethod
```

Comment:
This is an entire scripting interface. Maybe a separate PR would be more appropriate? Seems like you're doing multiple things here.